### ISSUE TITLE
Is it possible to get a count of notes with tags?

### ISSUE THREAD

**Issue by cvocvo** [jcgoette/baby_buddy_homeassistant#114]
Background commentary: https://github.com/orgs/babybuddy/discussions/666

Tl;dr is that I was asking if we could add some way to track the number of books read. @cdubz mentioned I can use tags and they've thought about graphing before; and he thinks this should be possible via the API. I think I could keep track of the number of books read (and probably read over time via a sensor) in HA if we could have sensors added or some way to add a custom sensor. The sensor(s) would just be a 'count' of the number of tags within a specified amount of time. Perhaps 7, 30, 365 days, and all time would be sufficient.

What do you think? This seems to align with some other feature requests as well that are accomplished via notes+tags in BabyBuddy. If you point me in the right direction I could try and look at the code as well.

Thank you!

**Comment by jcgoette** [comment]
Would seem doable. 

Related issues: https://github.com/jcgoette/baby_buddy_homeassistant/issues/58 and https://github.com/jcgoette/baby_buddy_homeassistant/issues/77.

**Comment by cvocvo** [comment]
> Would seem doable.
> 
> Related issues: #58 and #77.

Ahh yea I see you mention the same / a similar concept in this: https://github.com/jcgoette/baby_buddy_homeassistant/issues/77#issuecomment-1321225754
Would this existing PR accomplish what I'm thinking? https://github.com/jcgoette/baby_buddy_homeassistant/pull/78

**Comment by jcgoette** [comment]
I think it's necessary but not sufficient.
